src/averageAges.js

```diff
@@ -14,12 +16,15 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const arrayOfMan = (century)
+  ? people.filter(men => men.sex === 'm'
+    && Math.ceil(men.died / 100) === century)
+  : people.filter(men => men.sex === 'm');
```

Comment:
men - множественно число,
man - ед. ч.
(то же самое women/women - проверь код дальше)

```suggestion
  ? people.filter(man => man.sex === 'm'
    && Math.ceil(man.died / 100) === century)
  : people.filter(man => man.sex === 'm');
```